lab-allie/app/component/landing/login/login.html

```diff
@@ -0,0 +1,33 @@
+<section>
+  <form
+    name="loginForm"
+    ng-submit="loginCtrl.login()"
+    novalidate>
+      <div
+        ng-class="{
+          'error': loginForm.username.$invalid,
+          'success': loginForm.username.$valid
+        }">
+        <input type="text"
+          name="username"
+          placeholder="username"
+          ng-minlength="4"
+          ng-model="loginCtrl.user.username"
+          required>
+        <input type="text" placeholder="email">  
```

Comment:
You can get rid of this field. It's not really doing anything for you on the login front.